src/parameterizations/lateral/MOM_mixed_layer_restrat.F90

```diff
@@ -338,7 +349,21 @@ subroutine mixedlayer_restrat_OM4(h, uhtr, vhtr, tv, forces, dt, h_MLD, VarMix,
   h_neglect = GV%H_subroundoff
   if (CS%front_length>0.) then
     res_upscale = .true.
-    I_LFront = 1. / CS%front_length
+    do j=js-1,je+1 ; do i=is-1,ie+1
+      mle_fl_2d(i,j) = CS%front_length
+    enddo ; enddo
+  elseif (CS%front_length == 0. .and. CS%fl_from_file) then
+    res_upscale = .true.
+    call time_interp_external(CS%sbc_fl, CS%Time, mle_fl_2d, turns=G%HI%turns)
```

Comment:
This call to `time_interp_external` might need the argument `scale=US%m_to_L`.

It looks like the units in the comment in the declaration for `mle_fl_2d` might be wrong.
It says `[H ~> m or kg m-2]`, despite `mle_fl_2d` getting assigned `CS%front_lenght`
in another block, which has `scale=US%m_to_L` in its `get_param` call.